### PR TITLE
LPD-40406 

### PR DESCRIPTION
**Pull request by gianmarcobrunialti**
https://liferay.atlassian.net/browse/LPD-40406
https://liferay.atlassian.net/browse/LPD-40417
https://liferay.atlassian.net/browse/LPD-40422
https://liferay.atlassian.net/browse/LPD-40451

### CODE REVIEW

File: modules/apps/commerce/commerce-site-initializer/commerce-site-initializer/src/main/resources/site-initializer/layouts/01_catalog/product-detail/page-definition.json [hunk 1]
```diff
@@ -73,6 +73,17 @@
...
+							},
+							{
+								"definition": {
+									"widgetInstance": {
+										"widgetConfig": {
+										},
+										"widgetName": "com_liferay_commerce_product_content_web_internal_portlet_CPCompareContentMiniPortlet"
+									}
+								},
+								"id": "3fa3f597-b727-eb12-8265-67c6319ea5aa",
```
Comment (smottal) [inline modules/apps/commerce/commerce-site-initializer/commerce-site-initializer/src/main/resources/site-initializer/layouts/01_catalog/product-detail/page-definition.json:85]: is it correct to have the id?

File: workspaces/.idea/inspectionProfiles/Project_Default.xml
Comment (smottal) [inline workspaces/.idea/inspectionProfiles/Project_Default.xml:1]: mmmmm

**Review by smottal — CHANGES_REQUESTED** [review]

File: modules/apps/commerce/commerce-site-initializer/commerce-site-initializer/src/main/resources/site-initializer/layouts/01_catalog/product-detail/page-definition.json (hunk 1 above)
Comment (gianmarcobrunialti) [reply modules/apps/commerce/commerce-site-initializer/commerce-site-initializer/src/main/resources/site-initializer/layouts/01_catalog/product-detail/page-definition.json:85]: Nopes

File: workspaces/.idea/inspectionProfiles/Project_Default.xml
Comment (gianmarcobrunialti) [reply workspaces/.idea/inspectionProfiles/Project_Default.xml:1]: errr...

**Comment by gianmarcobrunialti** [comment]
ci:forward

**Comment by gianmarcobrunialti** [comment]
ci:forward:force

**Pull request closed without merging**